pkg/archive/archive.go

```diff
@@ -460,13 +460,19 @@ func (ta *tarAppender) addTarFile(path, name string) error {
 	//writing tar headers/files. We skip whiteout files because they were written
 	//by the kernel and already have proper ownership relative to the host
 	if !strings.HasPrefix(filepath.Base(hdr.Name), WhiteoutPrefix) && !ta.IDMappings.Empty() {
```

Comment:
Indeed, ok done

Still not familiar with whiteout notion, should I check it? Also I've not managed Lchown & GOOS here, do I need to take care?